01/app.js

```diff
@@ -0,0 +1,2 @@
+const btn = document.querySelector('button');
```

Comment:
Przy wyszukiwaniu elementów warto skorzystać ze zdarzenia `DOMContentLoaded`

Reply:
Chej, dzięki za wszystkie wskazówki. Mam wątpliwości czy warto korzystać z DOMContentLoaded, była o tym dyskusja na stack overflow (https://stackoverflow.com/questions/34807614/should-i-wait-for-domcontentloaded-event-if-i-place-a-script-tag-at-the-end-of-a). Czy ewentualnie mógłbyś podesłać jakieś artykuły, że jednak warto tego używać mimo, że link do skryptu mam na końcu body? Dzięki.